projects/saturn/src/main/java/io/fairspace/saturn/webdav/DirectoryResource.java

```diff
@@ -361,4 +364,4 @@ private void uploadMetadata(FileItem file) throws BadRequestException, ConflictE
             throw new BadRequestException("Error applying metadata", e);
         }
     }
-}
+}
```

Comment:
Why do you remove the newlines at the end of java files? Or is it your Intellij setup?  If so, please change it, those lines should be kept there